ui/StatusQ/src/StatusQ/Components/StatusMessage.qml

```diff
@@ -295,6 +295,45 @@ Control {
                             }
                         }
                     }
+
+                    Loader {
+                        active: root.messageDetails.contentType === StatusMessage.ContentType.Image && !editMode && messageDetails.album.length > 1
```

Comment:
Both loaders kinda do the same thing, except a difference on `messageDetails.album.length <> 1`, maybe you could just use a single Loader instead and change the `sourceComponent` depending on that condition instead.